src/game_api/entity_db.hpp

```diff
@@ -55,7 +55,9 @@ struct EntityDB
         };
         CollisionInfo default_collision_info;
     };
+    // MASK, will only call collision2 when colliding with entities that match this mask.
```

Comment:
You dropped some slashes on the way here.

Reply:
didn't realize it was supposed to be 3, should be good